style/sass/README.md

```diff
@@ -45,10 +45,9 @@
 * Use ID and class names that are as short as possible but as long as necessary.
 * Avoid using the direct descendant selector `>`.
 * Avoid nesting more than 3 selectors deep.
-* Use HTML tags on vague classes that need a qualifier like `header.application` not `.main`.
-* Avoid using the HTML tag in the class name: `section.news` not `section.news-section`.
-* Avoid using HTML tags on classes for generic markup `<div>`, `<span>`: `.widgets` not `div.widgets`.
-* Avoid using HTML tags on classes with specific class names like `.featured-articles`.
+* Avoid over-qualified selectors:
```

Comment:
I did want to stray away from “HTML tag qualifier”, because there are other ways in which a selector can be over-qualified. The “Do” and “Don't” was added to attempt to be very clear as to _what_ made that an over-qualified selector (having the `h1` in there).